scripts/installer-lang/ta.ps1

```diff
@@ -0,0 +1,49 @@
+[PSCustomObject]@{
+    Welcome         = "
+╔═══════════════════════════════════╗
+║ Windows SpotXக்கு வரவேற்கிறோம் ║
+╚═══════════════════════════════════╝"
+    Incorrect       = "அச்சச்சோ, தவறான மதிப்பு,"
+    Incorrect2      = "மீண்டும் உள்ளிடவும்"
+    Download        = "பதிவிறக்குவதில் பிழை"
+    Download2       = "5 வினாடிகளில் மீண்டும் வேண்டுகோள் விடுக்கப்படும்..."
+    Download3       = "மீண்டும் பிழை"
+    Download4       = "உங்கள் பிணைய அமைப்புகளைச் சரிபார்த்து, நிறுவலை மீண்டும் இயக்கவும்"
+    Download5       = "Spotifyயைப் பதிவிறக்குகிறது"
+    StopScript      = "Script நிறுத்தப்பட்டது"
+    MsSpoti         = "Spotifyயின் Microsoft Store பதிப்பு கண்டறியப்பட்டது, அதை ஆதரிக்கவில்லை"
+    MsSpoti2        = "Microsoft Store Spotify பதிப்பை நிறுவல் நீக்கவும் [Y/N]"
+    MsSpoti3        = "MS Spotifyயை தானாக நிறுவல் நீக்குகிறது..."
+    MsSpoti4        = "MS Spotifyயை நிறுவல் நீக்குகிறது..."
+    Prem            = "Premium கணக்கிற்கான மாற்றம்..."
+    OldV            = "Spotifyயின் காலாவதியான பதிப்பு கண்டறியப்பட்டது"
+    OldV2           = "உங்கள் Spotify {0} பதிப்பு காலாவதியானது, {1}க்கு மேம்படுத்த பரிந்துரைக்கப்படுகிறது"
+    OldV3           = "புதுப்பிக்க வேண்டுமா? [Y/N]"
+    AutoUpd         = "பரிந்துரைக்கப்பட்ட பதிப்பிற்கான தானியங்கி புதுப்பிப்பு"
+    DelOrOver       = "{0}யின் தற்போதைய பதிப்பை நிறுவல்நீக்கம் செய்ய விரும்புகிறீர்களா அல்லது நீங்கள் அதன் மீதே நிறுவ விரும்புகிறீர்கள்? Y [Uninstall] / N [Install Over]"
```

Comment:
here

Reply:
Combined to get meaning in tamil Meaning

Reply:
ok, if there are problems with substitution variables, let me know

Reply:
No issues